packages/synapse-react-client/src/lib/containers/auth/TwoFactorBackupCodes.tsx

```diff
@@ -59,8 +54,13 @@ export default function TwoFactorBackupCodes(props: TwoFactorBackupCodesProps) {
     if (!showReplaceOldCodesWarning) {
       generateCodes()
     }
+    // Run on mount only
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  const { visibleOnPrintClassName, hiddenOnPrintClassName } =
+    useMediaPrintOnly()
```

Comment:
Custom hook will hide everything except those elements where we apply `visibleOnPrintClassName`